src/client/src/components/icons/icons.stories.tsx

```diff
@@ -1,19 +1,68 @@
-import { storiesOf } from "@storybook/react"
-import React from "react"
-
 import * as icons from "./"
-import { ReactiveTraderIcon } from "./types"
-
-const stories = storiesOf("Components/Icons", module)
 
-type SVGElement = { type: "svg" } & JSX.Element
+export default {
+  title: "Components/Icons",
+  component: icons,
+}
 
-const isSvg = (T: SVGElement | React.FC<ReactiveTraderIcon>): T is SVGElement =>
-  (T as SVGElement)?.type === "svg"
+// stories can no longer be created dynamically as CSF are statically defined
```

Comment:
sob..

Reply:
let me have a tinker .. what is the blocker to creating all these in a loop and chucking something out in an alternative export stmt?